bigframes/core/blocks.py

```diff
@@ -102,15 +102,24 @@ class PandasBatches(Iterator[pd.DataFrame]):
     """Interface for mutable objects with state represented by a block value object."""
 
     def __init__(
-        self, pandas_batches: Iterator[pd.DataFrame], total_rows: Optional[int] = 0
+        self,
+        pandas_batches: Iterator[pd.DataFrame],
+        total_rows: Optional[int] = 0,
+        *,
+        total_bytes_processed: Optional[int] = 0,
```

Comment:
I wonder if this should default to `None` to indicate cases where query execution didn't happen?